boa/schemas/model.py

```diff
@@ -0,0 +1,97 @@
+# generated by datamodel-codegen:
+#   filename:  recipe.v1.json
+#   timestamp: 2021-11-22T15:42:45+00:00
+
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, Extra, Field, conint
+
+
+class Package(BaseModel):
+    class Config:
+        extra = Extra.forbid
+
+    name: Optional[str] = Field(None, description="The package name")
+    version: Optional[str] = Field(None, description="The package version")
+
+
+class SourceItem(BaseModel):
+    url: Optional[str] = None
+    sha256: Optional[str] = None
+    md5: Optional[str] = None
+    sha1: Optional[str] = None
+    patches: Optional[List[str]] = None
+    folder: Optional[str] = None
+    git_rev: Optional[str] = None
+    git_url: Optional[str] = None
+
+
+class Build(BaseModel):
+    number: conint(ge=0)
+    string: Optional[str] = None
+    run_exports: Optional[Dict[str, Any]] = None
+    skip: Optional[List[str]] = None
+    noarch: Optional[str] = None
+    pre_link: Optional[str] = Field(None, alias="pre-link")
+    post_link: Optional[str] = Field(None, alias="post-link")
+    pre_unlink: Optional[str] = Field(None, alias="pre-unlink")
+    ignore_run_exports: Optional[List[str]] = None
+    ignore_run_exports_from: Optional[List[str]] = None
+
+
+class Package1(BaseModel):
```

Comment:
can we merge this with the regular Package?

Reply:
yes, we can. done